src/ws-inline-edit/ws-inline-edit.js

```diff
@@ -0,0 +1,95 @@
+import {React, Component} from '../imports';
+require('./ws-inline-edit.scss');
```

Comment:
Have a look at ./demo/index.scss
We don't import scss files inside the js because not every system supports this but everyone got his own build of scss files in there projects.